Output.java

```diff
@@ -1,9 +1,16 @@
 public class Output {
```

Comment:
Why these changes are in that pull-request? It should not be there as it doesn't relate to DEV-3